src/libs/Navigation/AppNavigator/AuthScreens.tsx

```diff
@@ -576,7 +572,6 @@ function AuthScreens({session, lastOpenedPublicRoomID, initialLastUpdateIDApplie
                 </RootStack.Navigator>
                 <SearchRouter />
             </View>
-            {didPusherInit && <ActiveGuidesEventListener />}
```

Comment:
Why this was removed? Overall it relates to all updates in this file, was it a mistake during conflicts resolution?

Reply:
Probably yes 😅